cli/lib/cypress.js

```diff
@@ -50,6 +51,22 @@ const cypressModuleApi = {
       })
     })
   },
+
+  cli: {
+    /**
+     * Parses CLI arguments into an object that you can pass to "cypress.run"
+     * @example
+     *  const cypress = require('cypress')
+     *  const cli = ['cypress', 'run', '--browser', 'firefox']
+     *  const options = await cypress.cli.parseRunArguments(cli)
+     *  // options is {browser: 'firefox'}
+     *  await cypress.run(options)
+     * @see https://on.cypress.io/module-api
+     */
+    parseRunArguments (args) {
+      return cli.parseRunCommand(args)
+    },
```

Comment:
```suggestion
    parseRunArguments: cli.parseRunCommand(args),
```

Reply:
@flotwig read again your suggestion ;) It is not doing what you think it is doing